contracts/src/tests/create_card.cairo

```diff
@@ -0,0 +1,51 @@
+use core::traits::{Into, Default};
+use array::ArrayTrait;
+use serde::Serde;
+use option::{Option, OptionTrait};
+use debug::PrintTrait;
+use dojo::world::IWorldDispatcherTrait;
+
+use tsubasa::components::{Game, Roles, Card};
+use tsubasa::tests::utils::spawn_world;
+use tsubasa::systems::create_card_system;
+
+#[test]
+#[available_gas(30000000)]
+fn test_create_card() {
+    let player1 = starknet::contract_address_const::<0x1>();
+    let player2 = starknet::contract_address_const::<0x2>();
+
+    let world = spawn_world();
+
+    //Create Card For test prupose
+    let mut create_card_calldata: Array = ArrayTrait::new();
+    create_card_calldata.append(1); //Token_id
+    create_card_calldata.append(22); //Dribble
+    create_card_calldata.append(17); //Defense
+    create_card_calldata.append(10); //cost
+    create_card_calldata.append(3); //Role 0:Goalkeeper / 1:Defender etc..
+    create_card_calldata.append(0); //is_captain 0:false - 1:true 
+
+    world.execute('create_card_system', create_card_calldata);
+
+    let mut create_card_calldata_player2: Array<felt252> = ArrayTrait::new();
+    create_card_calldata_player2.append(2); //Token_id
+    create_card_calldata_player2.append(10); //Dribble
+    create_card_calldata_player2.append(15); //Defense
+    create_card_calldata_player2.append(10); //cost
+    create_card_calldata_player2.append(1); //Role 0:Goalkeeper / 1:Defender etc..
+    create_card_calldata_player2.append(0); //is_captain 0:false - 1:true 
```

Comment:
use the `array![]` macro

Reply:
[array![] macro fix](https://github.com/keep-starknet-strange/tsubasa/pull/69/commits/1c75992736d8e544201ddb8fe967849184806b18)